front/src/applications/stdcm/views/StdcmView.tsx

```diff
@@ -181,7 +181,8 @@ const StdcmView = () => {
   if (error && error.message !== NO_CONFIG_FOUND_MSG) throw error;
 
   return (
-    <div role="button" tabIndex={0} className="stdcm" onClick={() => setShowStatusBanner(false)}>
```

Comment:
Removing this has accessibility consequences: it's no longer possible to activate the button via the keyboard.

Reply:
Hm, but it's not really a button since it wraps the whole STDCM view… I think we should drop `role="button"` as well?

Reply:
Interacting with this div just removes the simulation banner. Do we need to activate it on the keyboard?

Reply:
The container doesn't act like a button, so I don't think `role="button"` is appropriate here.

There are many ways to "interact" with a form, using the mouse to do is is just one of them. For instance, the Tab key focuses the next form element, or the "Next" key on an on-screen keyboard (e.g. on an Android tablet) has the same effect.

Maybe `onFocus` is what we want here instead of `onClick`?